app/lib/frontend/templates/views/pkg/install_tab.mustache

```diff
@@ -16,28 +16,26 @@
 {{#use_as_a_library}}
 <h2>Use this package as a library</h2>
 <h3>1. Depend on it</h3>
-<p>Add this to your package's pubspec.yaml file:</p>
-<pre><code class="language-yaml">{{dependencies_key}}:
-  <strong>{{package}}: {{example_version_constraint}}</strong>
-</code></pre>
-<h3>2. Install it</h3>
-<p>You can install packages from the command line:</p>
+<p>Run this command:</p>
 {{#use_pub_get}}
-  <p>with pub:</p>
-  <pre><code class="language-shell">$ <strong>pub get</strong>
-</code></pre>
+<p>With Dart:</p>
+<pre><code class="language-shell"> $ dart pub add {{package}}</code></pre>
 {{/use_pub_get}}
 {{#use_flutter_packages_get}}
-  <p>with Flutter:</p>
-  <pre><code class="language-shell">$ <strong>flutter pub get</strong>
-</code></pre>
+<p>With Flutter:</p>
+<pre><code class="language-shell"> $ flutter pub pub add {{package}}</code></pre>
```

Comment:
Maybe we shouldn't show this part

Reply:
Yeah - maybe this is too early. We need `flutter pub add` to work first...